WordPressAuthenticatorTests/GoogleSignIn/CodeVerifier+Fixture.swift

```diff
@@ -0,0 +1,12 @@
+@testable import WordPressAuthenticator
+
+extension ProofKeyForCodeExchange.CodeVerifier {
+
+    /// A code verifier for testing purposes that is guaranteed to be valid and deterministic.
+    ///
+    /// The reason we care about it being deterministic is because we don't want implicit randomness test.
+    /// The only place were we want to use random values in the `CodeVerifier` tests which explicitly check the random generation.
+    static func fixture() -> Self {
+        .init(value: (0..<minimumLength).map { _ in "a" }.joined())!
```

Comment:
Another nit, but if we want to avoid `!` here (in case SwiftLint or something wants to yell at us for it) we could add a private initializer like `init(knownGoodValue:` that uses a `precondition` or similar to crash if this API is misused. 

Not sure if we care, but it's an option 🤷

Reply:
I'd like to park this suggestion for the moment. We don't have a rule to flag force unwraps in the test code ATM and there are arguments in favor of leaving this as an option, trading off the possibility of test-runtime crashes in case of developer error for more compact code.